paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
Of course your testing with emoji :wink: Seems to be centered already, at least for me on Chrome/Firefox.

![image](https://cloud.githubusercontent.com/assets/1319720/13022874/df1b460e-d198-11e5-80ab-3cba45b87935.png)